### PR TITLE
docs/variants/msi_z690/montage.md: add file

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki [kamil.pokornicki@3mdeb.com]

### CODE REVIEW

File: docs/variants/msi_z690/montage.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Dasharo compatible with MSI PRO Z690-A DDR4 - montage
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:1]: I think that we should change the name of this file, i. e. for something like this: `Presale device assembly and validation`

File: docs/variants/msi_z690/montage.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## Intro
+
+This document describes the montage procedure of the MSI PRO Z690-A DDR4 with
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:5]: Let's name this procedure not montage but assembly

File: docs/variants/msi_z690/montage.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+|:-------------|:---------------------------------------------------------------:|:---------------------------------------:|
+| Motherboard  | MSI PRO Z-690A DDR4                                             |![Motherboard](images/motherboard.jpg)   |
+| CPU          | Intel Core i5-12600K, 3.7G                                      |![CPU](images/cpu.jpg)                   |
+| Cooling      | Noctua CPU NH-U12S Redux (w/ Noctua NM-i17xx-MP78 Mounting Kit) |![Cooler](images/cpu-cooler.jpg)         |
+| RAM          | Kingston Fury Beast, DDR4, 4*8GB (32GB Total), 3600MHz, CL17    |![RAM](images/memory.jpg)                |
+| Power Supply | Seasonic Focus PX 750W 80 Plus Platinum                         |![Power Supply](images/power-supply.jpg) |
+| Storage      | SSD Intel 670p 512 GB M.2 2280 PCI-E x4 Gen3 NVMe               |![Storage](images/storage.jpg)           |
+| Enclosure    | SilentiumPC Armis AR1                                           |![Enclosure](images/enclosure.jpg)       |
+
+Additionally, you will need a regular Phillips screwdriver and thermal paste.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:20]: Do we need a Phillips screwdriver? Might we use a screwdriver from another producer? 

In this section, the type of screwdriver should be mentioned.

File: docs/variants/msi_z690/montage.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+Remember that make all connections in the grounding strap.
+
+## Montage
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:23]: Device assembly in my opinion sounds better.

File: docs/variants/msi_z690/montage.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    connection to the `G` and `H` slots and the cable marked RJ21 to connect the
+    `F` slots.
+
+    ![Power Supply](images/msi_z690_montage_power_supply.png)
+
+1. Connect the fan built into the enclosure to the indicated place:
+
+    ![Fan](images/msi_z690_montage_fan.jpg)
+
+1. Close the case and connect the computer to power. Start it with the button
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:111]: This section is so long. I think, that we should split it into several subsections, i. e. one section per one element.

File: docs/variants/msi_z690/montage.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+    A detailed description of the connection cable marked `B` can be found in
+    the [Board manual](https://download.msi.com/archive/mnu_exe/mb/PROZ690-AWIFIDDR4_PROZ690-ADDR4100x150.pdf)
+    in section: JFP1, JFP2: Front Panel Connectors.
+
+1. Connect the speaker to the slot marked before `E`:
+
+    ![Speaker](images/msi_z690_montage_speaker.jpg)
+
+1. Now is the time to screw the motherboard onto the case.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:99]: We should avoid formatting phrases like this. Please describe only the activities.

File: docs/variants/msi_z690/montage.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+    ![Power Supply](images/msi_z690_montage_power_supply.png)
+
+1. Connect the fan built into the enclosure to the indicated place:
+
+    ![Fan](images/msi_z690_montage_fan.jpg)
+
+1. Close the case and connect the computer to power. Start it with the button
+    on the front panel. If all connections have been made correctly, the
+    computer will start up.
+
```
Comment (sulewskiprzemyslaw) [inline docs/variants/msi_z690/montage.md:114]: And if the device starts, what will be further? Maybe in this place, we should add information, that Dasharo logging screen should appear after a while? Maybe also we should add information about booting to any system (live CD), to check if everything is OK?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/montage.md (hunk 3 above)
Comment (macpijan) [reply docs/variants/msi_z690/montage.md:20]: `Phillips` is a type, not vendor. See: `Phillips, generically called "cross-recess", "cross-head", or "cross-point". ` https://en.wikipedia.org/wiki/Screwdriver

File: docs/variants/msi_z690/montage.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: renamed

File: docs/variants/msi_z690/montage.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: renamed

File: docs/variants/msi_z690/montage.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: I think that as it is good, I haven't updated it

File: docs/variants/msi_z690/montage.md (hunk 4 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: renamed

File: docs/variants/msi_z690/montage.md (hunk 5 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: I splitted this section

File: docs/variants/msi_z690/montage.md (hunk 6 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: OK, I updated it

File: docs/variants/msi_z690/montage.md (hunk 7 above)
Comment (Pokisiekk) [reply docs/variants/msi_z690/montage.md]: I added shorts steps

File: docs/variants/msi_z690/montage.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/msi_z690/montage.md]: @macpijan ok, thank You for the information

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**